pandas_hw/pandas_hw.py

```diff
@@ -0,0 +1,367 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[3]:
+
+
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+import seaborn as sns
+import re
+from matplotlib.patches import ConnectionPatch
+
+
+# ## Working with real data
+
+# ### read_gff and read_bed6 functions for reading the corresponding formats.
+
+# In[2]:
+
+
+def read_gff(file):
+    return pd.read_table(file , sep='\t', skiprows=1, 
+                  names = ['chromosome', 'source', "type", "start", "end", "score", "strand", "phase", "attributes"])
```

Comment:
Твой код учитывает только случай, где комментарий занимает только первую строку. Однако чаще всего в файлах данного формата могут быть коментарии на ммного строк, их нужно учитывать. Благо в пандасе это делается супер просто
```suggestion
    return pd.read_table(file , sep='\t', comment="#", 
                  names = ['chromosome', 'source', "type", "start", "end", "score", "strand", "phase", "attributes"])
```